packages/compass-components/src/components/virtual-grid.tsx

```diff
@@ -0,0 +1,287 @@
+/* eslint-disable react/prop-types */
```

Comment:
I think I asked this before in another pr and can't quite remember the answer (sorry!), by why do we need this?